### DISCUSSION TITLE
Not able to enforce https

### DISCUSSION THREAD

**Discussion by JINU98** [community/community#118335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hii, I am unable to enforce https on our website: aiisc.ai. The DNS check works fine, but it shows the following issue : 
<img width="804" alt="Screenshot 2024-04-08 at 4 16 31 PM" src="https://github.com/community/community/assets/44402998/9d14b2b2-b4bf-488a-832f-31b37555d5b1">
TLS certificate is being provisioned. This may take up to 15 minutes to complete.

**Comment by jfullstackdev** [comment]
is it already more than 24 hours ? you need to wait that long, Copilot confirmed it too, here is its answer

--- 

When you enable HTTPS for your GitHub Pages site, GitHub automatically requests a TLS certificate from Let's Encrypt, a free certificate authority. This certificate is necessary to establish a secure connection between your site and its visitors.

The message "certificate has not finished being issued" means that the process of obtaining this certificate is still ongoing. This process can take some time, usually up to 24 hours, but sometimes longer.

During this time, you might not be able to access your site over HTTPS, and you might see security warnings in your browser. Once the certificate has been issued and installed, your site will be accessible over HTTPS without any warnings.

If the process takes longer than 24 hours, or if you see this message for an extended period of time, it might indicate a problem with the certificate issuance process. In this case, you should contact GitHub Support for assistance.
